harvardcards/templates/deck_view.html

```diff
@@ -108,7 +108,7 @@ <h3 data-editable="yes" data-editable-id="{{deck.id}}" title="Click to edit titl
                     </div>
                     <div class="hideRevealHr">
                         {% if is_quiz_mode %}
-                        <a href="javascript:void(0)" class="adminMenuBtn reveal">Reveal</a>
+                        <button type="button" class="adminMenuBtn reveal" accesskey="R">Reveal</button>
```

Comment:
It is not. In fact, invoking it often involves a `shift`. All examples that I saw used an uppercase.